telegram/message.py

```diff
@@ -125,6 +129,7 @@ def __init__(self,
                  sticker=None,
                  video=None,
                  voice=None,
+                 video_note=None,
```

Comment:
Same note like I left on #628 - should new parameters be added in the middle or in the end of the function?